libnavigator/src/test/resources/multileg_route.json

```diff
@@ -0,0 +1 @@
 arrived at your destination","type":"arrive"},"voiceInstructions":[],"bannerInstructions":[],"driving_side":"right","weight":0.0,"intersections":[{"location":[-77.153468,38.77091],"bearings":[21],"entry":[true],"in":0}]}],"annotation":{"distance":[11.242777252846174,11.706862088989638,15.21167162421484,13.267329797617013,10.882000795714738,7.756958787918375,5.615804188656081,69.74181988251104,118.80802507531325,41.94640060670711,11.139179363750046,49.457578312479654,15.130752607930281,9.617469930149234,10.742154057699196,56.471916253989484,27.506722989807894,26.323723350288706,205.2184701501639,45.56017941065718,32.7303273037294,35.55989486600731,41.10720840101533,23.957924983189184,25.808920972586215,73.64857698356425,44.77339489823673,138.8682648528186,18.742463709516244,51.02417900300925,20.010113706516375,34.712074255405284,124.0685296965995,22.46855463641897,35.63870677624317,19.72370567191629,45.63799217654485,35.10495613093475,2.594681025749277,106.76569226561834,20.77486597131294,35.93393804603291,78.3406801773675,129.0798186942903,11.717647780627496,23.926385300173756,14.58714931455207,20.729953263766536,7.887689681480181,6.240589398931153,13.716188146849971,31.4746247126341,6.679023987995285,43.938529758566276,82.60512616921073,67.75488010827317,22.336715083610425,27.4775894542259,35.32221364505742,16.761929414932776,9.02997942912682,23.6832839488985,15.775411045447989,45.93942323995397,32.816135460485725,21.525538999149465,11.149374959100331,11.404123953381957,21.59187900294635,22.683562777114748,20.150628497151523,1.671303515405502],"duration":[1.3,1.3,1.7,1.5,1.2,0.9,1.0,12.0,14.3,4.3,3.3,14.8,4.5,0.5,0.5,2.7,1.3,1.2,10.1,2.2,1.6,1.7,2.1,1.2,1.3,4.0,2.4,8.1,1.2,3.4,1.3,1.7,6.2,2.2,3.6,2.0,6.1,2.7,0.2,8.4,1.6,3.3,7.2,9.7,0.9,1.8,1.1,1.6,0.6,0.5,1.0,2.4,0.4,2.8,24.8,7.0,2.3,2.0,2.5,1.2,0.7,1.6,1.2,3.6,2.6,2.3,1.2,1.2,2.3,3.3,2.9,0.2],"congestion":["low","low","low","low","low","low","unknown","unknown","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","low","moderate","low","low","low","low","moderate","moderate","low","low","low","low","low","low","moderate","low","low","low","low","severe","heavy","heavy","low","low","low","low","low","low","low","low","low","low","low","low","unknown","unknown","unknown"]}}],"routeOptions":{"baseUrl":"https://api.mapbox.com","user":"mapbox","profile":"driving-traffic","coordinates":[[-77.1576396,38.7830304],[-77.1670888,38.7756155],[-77.1534183,38.7708948]],"alternatives":false,"language":"en","continue_straight":false,"roundabout_exits":false,"geometries":"polyline6","overview":"full","steps":true,"annotations":"congestion,distance,duration","voice_instructions":true,"banner_instructions":true,"voice_units":"imperial","access_token":"pk.eyJ1IjoiY2FmZXNpbGVuY2lvIiwiYSI6ImNrN2R3bnNxZDBkaTMzbm52bHJ2NmR4Z3YifQ.4ZUCu5Dqsyb_RfafMWe5tQ","uuid":"cka476dyl01qv7ak7ogvqdxgn"},"voiceLocale":"en-US"}
```

Comment:
nit could you format json for reading 🙇

Reply:
Intentionally not formatted to decrease the size of the file and ultimately the size of the repo :)

Reply:
I've checked and it's 25 vs 50 kb :trollface: . I'm ok to save space))

Reply:
All I see is that it's twice the size :D